Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/ApiPropertyMappingEnhancer.ts

```diff
@@ -37,18 +39,21 @@ function parentPrefixRemovalConvention(property: EntityProperty): string {
  * API descriptor reference property names are suffixed with "Descriptor"
  */
 function apiDescriptorReferenceName(property): string {
-  return normalizeDescriptorSuffix(uncapitalize(property.fullPropertyName));
+  return normalizeDescriptorSuffix(uncapitalize(property.metaEdName));
```

Comment:
No rolename prefix on descriptor references e.g. Assessment.AssessedGradeLevelDescriptor -> gradeLevelDescriptor

Reply:
Another surprising error (`local.33b.http` line 466): posting to `studentSchoolAssociations`:
```json
{
  "message": [
    "gradeLevelDescriptor is required",
    "entryGradeLevelDescriptor is not allowed"
  ]
}
```

Reply:
Fixed - unit test added